### PR TITLE
pep8; removed unused vars

### PR DESCRIPTION
**Pull request by aledista**

### CODE REVIEW

**Comment by kastiglione** [comment]
Thanks for the pull request! Was this done using a tool? Could you update the pull request to leave out all the indentation, whitespace, and line wrapping changes? It would be great to remove unused vars, but I'd rather avoid reformatting the entire code base.

**Comment by aledista** [comment]
I changed my commit. I left out style conventions and removed unused imports, vars and fixed a couple of minor issues. I generally use PyCharm to identify issues like that.

File: commands/FBFlickerCommands.py [hunk 1]
```diff
@@ -108,11 +106,6 @@ def inputCallback(self, input):
       self.setCurrentView(v, oldView)
     elif input == 'p':
       recusionName = 'recursiveDescription'
-      isMac = runtimeHelpers.isMacintoshArch()
-
-      if isMac:
-        recursionName = '_subtreeDescription'
```
Comment (kastiglione) [inline commands/FBFlickerCommands.py]: This block shouldn't be removed. `recursionName` is used below.

File: fblldb.py [hunk 2]
```diff
@@ -129,13 +127,12 @@ def helpForCommand(command, filename):
...
     for option in command.options():
 
-      optionFlag = ''
       if option.longName and option.shortName:
         optionFlag = option.longName + '/' + option.shortName
       elif option.longName:
         optionFlag = option.longName
       else:
-        optionFlag = optiob.shortName
+        optionFlag = option.shortName
```
Comment (kastiglione) [inline fblldb.py]: :joy:

**Comment by aledista** [comment]
Readded ;)

**Comment by kastiglione** [comment]
Thanks again for the pull request!

**Comment by aledista** [comment]
You're welcome!

**Pull request merged**